src/Uno.UI.RuntimeTests/Tests/HotReload/Given_HotReloadWorkspace.cs

```diff
@@ -97,7 +97,7 @@ public async Task Initialize()
 	}
 
 	[TestCleanup]
-	public async Task TestCleanupWrapper()
+	public void TestCleanupWrapper()
```

Comment:
@Youssef1313 The original intent of this change was to ensure that kill was not a fire-and-forget. What's behavior with Task?

Reply:
In that case, I think we will need to call `WaitForExit`?

Reply:
Maybe, just to be sure. What was the issue originally?

Reply:
When tests fail and are re-tried, there is a race where the re-try *may* fail saying file is used by another process. Hoping it doesn't happen now that we wait for the process to exit.